faq.html

```diff
@@ -171,12 +167,12 @@ <h1 id="how-can-i-keep-myself-safe">How much privacy does Pixelated provide?</h1
   <section>
     <h1 id="sounds-good-how-can-i-contribute">Sounds good! How can I contribute?</h1>
 
-    <p>We know that it can still be a bit tough to set up your Pixelated environment and find an issue that&#39;s a good fit for your skills, and we&#39;re always looking for ways to lower the entry barrier for new contributors. If you hit any roadblock, you have good ideas, or just want to share your thoughts about onboarding etc., contact our community at <a href="mailto:community@pixelated-project.org" target="_blank">community@pixelated-project.org</a> so we can resolve these issues for you and everyone else. We also have an <a href="irc://irc.freenode.net/pixelated">IRC channel</a>.</p>
+    <p>We know that it can still be a bit tough to set up your Pixelated environment and find an issue that&#39;s a good fit for your skills, and we&#39;re always looking for ways to lower the entry barrier for new contributors. If you hit any roadblock, you have good ideas, or just want to share your thoughts about onboarding etc., contact our community at <a href="mailto:community@pixelated-project.org" target="_blank">community@pixelated-project.org</a> so we can resolve these issues for you and everyone else.
```

Comment:
Good question. Mostly because IRC requires activity, which I don't think we can guarantee. We need to decide if there will be any communication channels at all. Thoughts @olabini